### PR TITLE
Feat: implement label audio feature.

### LINKED ISSUE #8: Multiple calls to the API on clicking translate
Screenshot from 2020-02-28 14-35-05
The modal doesn't disappear after a single click, and on multiple clicks, a series of "undefined" results appear(P.S.. I got this after checking out @rkpattnaik780's PR locally).

### PR DESCRIPTION
**Pull request by rkpattnaik780**
This PR provides audio to the labels dynamically. Handles bugs related to receiving a response and toggling of the translate button. The audio and text translation buttons have been separated.

Fixes #8 

To Dos Left: 
- [ ] Redesign the popup
- [X] Handle cases where we toggle translation when audio is enabled.
- [X] Change the directory structure, the extension files should be moved to `src/`
- [ ] Add a pre-commit hook for linting stuffs (husky).
- [X] Add README and License.
- [ ] Add a CONTRIBUTING.md.
- [ ] Add the audio form input feature.

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  for (let i = 0, l = labels.length; i < l; i++) {
+    labels[i].innerText = output_labels[i];
+  }
+  if (isAudioEnabled) {
+    disableAudio();
+    enableAudio();
+  }
+};
+
+const trandlateLabels = () => {
```
Comment (manaswinidas) [inline src/content.js]: Just a small typo here, I guess: trandlateLabels --> translateLabels

File: src/content.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        labels[i].innerText = translatedLabels[i];
+      }
+
+      if (isAudioEnabled) {
+        disableAudio();
+        enableAudio();
+      }
+    },
+  });
+};
```
Comment (manaswinidas) [inline src/content.js]: This looks repetitive

File: src/content.js (hunk 1 above)
Comment (rkpattnaik780) [reply src/content.js]: This was a typo.

File: src/content.js (hunk 2 above)
Comment (rkpattnaik780) [reply src/content.js]: This block of code has been placed inside the callback to handle the case wherein the labels are translated and audio is enabled. This block changes the label audio according to the label language.

File: src/popup.html [hunk 3]
```diff
@@ -28,6 +28,7 @@
     <h4>Do you want to translate?</h4>
     <div>
       <button class="btn btn-primary" id="yes">Yes</button>
+      <button class="btn btn-primary" id="audio">Audio</button>
```
Comment (agarwalpooja) [inline src/popup.html]: Here can we change the id name to "translate" instead of "yes"?

File: mock_web/static/index.html [hunk 4]
```diff
@@ -36,45 +21,30 @@ <h2>Horizontal form</h2>
       <div class="col-sm-8">
         <input type="text" class="form-control" id="first-name" placeholder="Enter your first name" name="firstname">
       </div>
-      <div class="col-sm-2">
-          <input id="play" onclick='responsiveVoice.speak("पहला नाम");' type='button' value='🔊 Play'/>
-          </div>
     </div>
     <div class="form-group">
       <label class="control-label col-sm-2" for="email1">Last Name:</label>
```
Comment (agarwalpooja) [inline mock_web/static/index.html]: Can remove the for tag.

File: mock_web/static/index.html [hunk 5]
```diff
@@ -36,45 +21,30 @@ <h2>Horizontal form</h2>
...
       <div class="col-sm-8">
         <input type="text" class="form-control" id="last-name" placeholder="Enter your last name" name="lastname">
       </div>
-      <div class="col-sm-2">
-          <input id="play1" onclick='responsiveVoice.speak("अंतिम नाम");' type='button' value='🔊 Play'/>
-          </div>
     </div>
     <div class="form-group">
       <label class="control-label col-sm-2" for="email2">Email:</label>
```
Comment (agarwalpooja) [inline mock_web/static/index.html]: Can remove the for tag.

File: mock_web/static/index.html [hunk 6]
```diff
@@ -36,45 +21,30 @@ <h2>Horizontal form</h2>
...
       <div class="col-sm-8">
         <input type="email" class="form-control" id="email" placeholder="Enter email" name="email">
       </div>
-      <div class="col-sm-2">
-          <input id="play2" onclick='responsiveVoice.speak("ईमेल");' type='button' value='🔊 Play'/>
-          </div>
     </div>
     <div class="form-group">
       <label class="control-label col-sm-2" for="email3">Date of Birth:</label>
```
Comment (agarwalpooja) [inline mock_web/static/index.html]: Can remove the for tag.

File: mock_web/static/index.html [hunk 7]
```diff
@@ -36,45 +21,30 @@ <h2>Horizontal form</h2>
...
       <div class="col-sm-8">
         <input type="date" class="form-control" id="dob" placeholder="Enter your birth date" name="DOB">
       </div>
-      <div class="col-sm-2">
-          <input id="play3" onclick='responsiveVoice.speak("जन्म तिथि");' type='button' value='🔊 Play'/>
-          </div>
     </div>
     <div class="form-group">
       <label class="control-label col-sm-2" for="email4">Mobile Number:</label>
```
Comment (agarwalpooja) [inline mock_web/static/index.html]: Can remove the for tag.

File: src/popup.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
+document.addEventListener("DOMContentLoaded", function() {
+  var translateButton = document.getElementById("yes");
```
Comment (agarwalpooja) [inline src/popup.js]: Here can we change the id name to a meaningful name like "translate"

File: mock_web/static/index.html (hunk 4 above)
Comment (rkpattnaik780) [reply mock_web/static/index.html]: Have changed the value of for attribute  to the id it should point to.

**Pull request merged**